packages/ui/src/components/Box/Box.tsx

```diff
@@ -37,7 +40,11 @@ export const Box = React.forwardRef<unknown, BoxProps>(
 		}
 
 		return React.createElement(as, {
-			className: clsx([sprinkles(sprinklesProps), userClasses]),
+			className: clsx([
+				sprinkles(sprinklesProps),
+				userClasses,
+				props.hiddenScroll ? hiddenScroll : null,
```

Comment:
@ccschmitz Is this the right way to implement `hiddenScroll`?

Reply:
My rule of thumb for adding something to the sprinkles classes or `Box` component has been to use something twice, and on the third time support it at this level. Maybe this is something we will need to deal with a number of places throughout the app, but I wonder if we should keep this isolated the sourcemap error details UI for now, or just make it a general utility class that can be `import`ed as needed rather than expanding the surface area of the `Box` props.

No strong opinions here, BTW. Cool with me if you want to merge this as is!

Reply:
MacOS hides these scrollbars by default, but as the Windows Guy™, I use this snippet pretty frequently.